doc/user_guide/dynamo0p3.rst

```diff
@@ -848,7 +848,7 @@ types.
    an ``integer``-valued field as an argument.
 
 .. _lfric-no-cma-mdata-rules:
-   
+
 Rules specific to General-Purpose Kernels without CMA Operators
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
```

Comment:
I've realised that the introductory paragraph to the `Kernel` section:
![image](https://github.com/stfc/PSyclone/assets/17409212/ed41acd7-716c-4520-bc1c-730e60a55465)
needs updating as we are going to add support for a new kernel type here. When you do that, please could you also add a note that support for this kernel type is to be implemented (and reference the issue). Please also do that in the two new sections you've added about the rules for this kernel type.